src/models/settlement/facade.js

```diff
@@ -207,15 +209,8 @@ const settlementTransfersPrepare = async function (settlementId, transactionTime
           createdDate: transactionTimestamp
         })
         .transacting(trx)
-
-      if (doCommit) {
-        await trx.commit
-      }
     } catch (err) {
       Logger.isErrorEnabled && Logger.error(err)
-      if (doCommit) {
```

Comment:
perhaps get rid of `doCommit` variable if it's no longer used?

Reply:
done